cmd/goal/interact.go

```diff
@@ -477,6 +481,9 @@ var appExecuteCmd = &cobra.Command{
 	Short: "Execute a procedure on an application",
 	Args:  cobra.MinimumNArgs(1),
 	Run: func(cmd *cobra.Command, args []string) {
+		reportWarnf("app interact is deprecated and will be removed soon. Please speak up if the feature matters to you.")
```

Comment:
I'd add a sentence directing the end user to the function(s) they should use instead.

Reply:
I don't know? This is so outdated nobody is using it. Maybe some sort of ARC4 method invocation?